penelope_aerospace_pl_msgs/msg/ResultCodes.msg

```diff
@@ -10,7 +10,8 @@ string SUCCES=""
 uint16 RC_SUCCES=1
 
 # Generic failure message
-string FAILED="Action/Service failed.. See module log for more details."
+string FAILED_BY_USER="Action/Service failed scan, terminated by operator."
+string FAILED="Action/Service failed scan with errors."
 uint16 RC_FAILED=2
 
```

Comment:
The `FAILED` string and `RC_FAILED` are fallback options for others too. I would create specific result codes and messages for TNO:

```
# DVM status messages (Range between 100 and 199)
string DVM_FAILED_BY_USER="DVM scan failed, terminated by operator."
uint16 RC_DVM_FAILED_BY_USER=100

string FAILED_BY_USER="Action/Service failed scan, terminated by operator."
```

Reply:
I updated the code in the penelope_TNO_module. Now it's using the RC_DVM_FAILED_BY_USER.